docs/rewards_fees.md

```diff
@@ -14,22 +14,59 @@ All the participants of the Subspace Network are compensated for the work they d
+
+Initially, farmers receive exactly 0.1 tSSC for the block rewards for the blocks they propose and 0.1 tSSC for votes they submit. Subspace implements a decay function, that will gradually reduce these rewards every block as the chain progresses. Over the long-term, the decrease follows the exponential decay:
+
+$$\text{reference\_subsidy}=\text{initial\_subsidy}*e^{-\text{initial\_subsidy}*(n-\text{decay\_block\_start})/\text{max\_issuance\_tokens}}$$
+
+where $\text{initial\_subsidy}=0.1$ tSSC per block, $n$ is current block height, $\text{decay\_block\_start}=TBA$ is the block when the decay function was activated, and $\text{max\_issuance\_tokens}=100 000 000$ tSSC is the total number of credits to be ever issued by the Gemini-3h testnet protocol for this reward. Both block proposer rewards and vote rewards are computed using the same formula.
+
+This smooth reduction allows for higher rewards for early adopters, gradual increase of the circulating supply in a more controlled manner and an extended lifetime of issuance for the long-term viability of the chain.
+
+On Gemini-3h, the reference subsidy issuance is expected to decay following the curve below. For example, for the first 1 296 000 blocks (~90 days) it starts at 0.1 tSSC per block and decreases as follows:
+
+<div align="center">
+    <img src="/img/Gemini3h_Issuance_Decay-light.svg#gh-light-mode-only" alt="Gemini3h_Issuance_Decay" />
+    <img src="/img/Gemini3h_Issuance_Decay-dark.svg#gh-dark-mode-only" alt="Gemini3h_Issuance_Decay" />
+</div>
+
+Apart from the reference subsidy issuance decay, the rewards for the block proposer are also dynamic based on the demand for blockspace. The subsidy presented above is a reference for the amount issued for an empty block. However, if the chain is seeing blockspace utilization, block proposers are also earning through transaction fees to cover the farming costs, and the protocol will lower the reward issued to the proposer. This allows the protocol to conserve tokens for later issuance in periods of lower demand. 
+
+The demand for blockspace is measured as exponential moving average of the percentage of the maximum blockspace that is used by the normal transactions over the last 100 blocks, excluding operational transactions like votes and fraud proofs. 
+
+$$\text{blockspace\_utilization} = \frac{\sum encoded\ transaction\ size}{3.75 \text{MiB}}$$
+
+The final formula for the block proposer reward is:
+
+$$\text{proposer reward}=\text{reference\_subsidy}-\min(\text{reference\_subsidy},\text{max\_block\_fees})*\text{blockspace\_utilization}$$
+
+For each block, the protocol decides whether the current cost of storage and demand are high enough to cover the proposer's costs of farming that block. It will lower the issued reference subsidy slightly if fees are not sufficient, and lower significantly if fees are high.
```

Comment:
> It will lower the issued reference subsidy slightly

As in reduces by a value approaching 0? I had to read this a couple of times to try and understand.

Reply:
Yes, consider reference subsidy is 5, utilization is 25% and fees are 2:
`5-min(5,2)*0.25=5-0.5=4.5`
However if fees are 6:
`5-min(5,6)*0.25=5-1.25=3.75`